### DISCUSSION TITLE
How to manage product price in admin Dashboard

### DISCUSSION THREAD

**Discussion by mister-loop** [medusajs/medusa#4495] [Q&A] [answered]
I was adding products to my dashboard and noticed there was no price section. I then checked the frontend but the prices were displayed there. On some research, found that they were hard-coded into the seeder file. 

1. My use-case is that I only want to add products without any variants and options and hence with a single price. I came across this article which has a nice widget for prices and was wondering how to add this
https://medusajs.com/blog/medusa-admin-dashboard/
![image](https://github.com/medusajs/medusa/assets/89638794/96a9b8a4-154b-4688-acf4-9c0904819b12)

2. I looked at the price list section and added some products to a new list. Now there is a `price_list` table in db but no link table to map a list to a product. I traversed through all the tables but couldn't find the prices I made to the list.

**Comment by srindom** [comment, marked as answer]
There are two ways to manage product prices through admin:

1. Products > [Click a Product] > Variants > Edit Variant > Prices
  Here you can edit the price for the variant in each currency and region you have configured for the store.
2. Pricing > Add price list
  Here you can set custom prices for products. E.g., if you have a sale you can upload prices for your sale items and configure the sale's start and end dates. You can also scope price lists to certain customer groups e.g., to give VIP or wholesale customers special prices.

**Comment by kaisiant** [comment]
Hi, I just want to clarify.
For option 1, is the variant required to set the price? My products do not have any variants, does that mean I need to create a dummy default variant for all my products?

**Reply by hassanphp** [reply]
Yes, you must need to create variant.
